script/deploy/holesky/M2_Deploy_From_Scratch.s.sol

```diff
@@ -0,0 +1,207 @@
+            address(new TransparentUpgradeableProxy(address(emptyContract), address(eigenLayerProxyAdmin), ""))
+        );
+
+        // Deploy EigenPod Contracts
+        eigenPodImplementation = new EigenPod(
+            IETHPOSDeposit(ETHPOSDepositAddress),
+            delayedWithdrawalRouter,
+            eigenPodManager,
+            EIGENPOD_MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR,
+            EIGENPOD_GENESIS_TIME
+        );
+
+        eigenPodBeacon = new UpgradeableBeacon(address(eigenPodImplementation));
+        avsDirectoryImplementation = new AVSDirectory(delegationManager);
+        delegationManagerImplementation = new DelegationManager(strategyManager, slasher, eigenPodManager);
+        strategyManagerImplementation = new StrategyManager(delegationManager, eigenPodManager, slasher);
+        slasherImplementation = new Slasher(strategyManager, delegationManager);
+        eigenPodManagerImplementation = new EigenPodManager(
+            IETHPOSDeposit(ETHPOSDepositAddress),
+            eigenPodBeacon,
+            strategyManager,
+            slasher,
+            delegationManager
+        );
+        delayedWithdrawalRouterImplementation = new DelayedWithdrawalRouter(eigenPodManager);
+
+        // Third, upgrade the proxy contracts to point to the implementations
+        IStrategy[] memory initializeStrategiesToSetDelayBlocks = new IStrategy[](0);
+        uint256[] memory initializeWithdrawalDelayBlocks = new uint256[](0);
+        // AVSDirectory
+        eigenLayerProxyAdmin.upgradeAndCall(
+            TransparentUpgradeableProxy(payable(address(avsDirectory))),
+            address(avsDirectoryImplementation),
+            abi.encodeWithSelector(
+                AVSDirectory.initialize.selector,
+                executorMultisig, // initialOwner
+                eigenLayerPauserReg,
+                AVS_DIRECTORY_INIT_PAUSED_STATUS
+            )
+        );
+        // DelegationManager
+        eigenLayerProxyAdmin.upgradeAndCall(
+            TransparentUpgradeableProxy(payable(address(delegationManager))),
+            address(delegationManagerImplementation),
+            abi.encodeWithSelector(
+                DelegationManager.initialize.selector,
+                executorMultisig, // initialOwner
+                eigenLayerPauserReg,
+                DELEGATION_MANAGER_INIT_PAUSED_STATUS,
+                DELEGATION_MANAGER_MIN_WITHDRAWAL_DELAY_BLOCKS,
+                initializeStrategiesToSetDelayBlocks,
+                initializeWithdrawalDelayBlocks
+            )
+        );
+        // StrategyManager
+        eigenLayerProxyAdmin.upgradeAndCall(
+            TransparentUpgradeableProxy(payable(address(strategyManager))),
+            address(strategyManagerImplementation),
+            abi.encodeWithSelector(
+                StrategyManager.initialize.selector,
+                executorMultisig, //initialOwner
+                executorMultisig, //initial whitelister
```

Comment:
I think I mentioned this above, but I think this should be the 'ops' multisig (at least to match Goerli/mainnet configs)?

Reply:
ahh missed this, good catch

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/473/files#diff-dbf2ec4102ab841c3a08fabf952cdfe8d391f2b48cef41f0e29e5e6740be800aR132